examples/rf.py

```diff
@@ -110,21 +110,18 @@ def rfr(cfg, seed):
                      "memory_limit": 1024,
                      })
  
-# register function to be optimize
-taf = ExecuteTAFuncDict(rfr)
- 
 # example call of the function
 # it returns: Status, Cost, Runtime, Additional Infos
-def_value = taf.run(cs.get_default_configuration())[1]
+def_value = rfr(cs.get_default_configuration(), 1)
 print("Default Value: %.2f" % (def_value))
  
 # Optimize
 smac = SMAC(scenario=scenario, rng=np.random.RandomState(42),
-            tae_runner=taf)
+            tae_runner=rfr)
 try:
     incumbent = smac.optimize()
 finally:
     incumbent = smac.solver.incumbent
 
-inc_value = taf.run(incumbent)[1]
+inc_value = rfr(incumbent, 1)
```

Comment:
If you run the rfr directly without a TAE class,
the call will have not limits. Therefore, the `inc_value` will not be necessarily the same as SMAC has observed it.
I think this will not be an issue for the example, but it will be a bad example for others.